config/elasticsearch.php

```diff
@@ -0,0 +1,93 @@
+<?php
```

Comment:
we can have the configuration inside scout.php I would like to have every driver config in the same scout config file.